client/src/components/QA/QACardAnswers.jsx

```diff
@@ -72,16 +77,24 @@ const QACardAnswers = ({ currentAnswers }) => {
           <div className="qa-div-answers" ><b>A:</b> {answer.body}</div>
           <p className="qa-footer"> by {answer.answerer_name}, {format((new Date(answer.date)), "MMMM dd, yyyy")}
             <span className="qa-divider">|</span>
-            <span onClick={() => APICallHelpful(answer.id)}>Helpful? ({answer.helpfulness})</span>
+            <span onClick={() => {
```

Comment:
TO use event default, I think you will need to add `event` to the anonymous function parameters.
```suggestion
            <span onClick={(event) => {
```